README.md

```diff
@@ -47,21 +71,30 @@ This will be the file that holds your S3 and SQS configurations.
       * a first-time login with your GitHub credentials
       * in the Settings -> Applications -> federalist view in your GitHub Account
 1. Type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal.
-1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.
-1. Enter your passcode back into terminal.
-1. Type `cf apps`.
+1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.  You'll need to login using your cloud.gov credentials.
+1. Enter your passcode back into terminal.  (Note: if you have a problem at this point it's likely because you don't have permissions.  [Contact us] (federalist-support@gsa.gov) for help.)
+1. Type `cf apps`.  This will list the current, running applications. 
```

Comment:
This isn't really necessary, just perhaps helpful. Maybe strike it.

Reply:
I included this because I wondered if it might be a bit frustrating for someone to get all the way to this point, and then not have cloud.gov credentials.  Perhaps we should add it as a requirement before they start above?  The permissions were an issue I ran into - if this is unusual, we could strike this.

Reply:
I meant the whole step of running `cf apps`.

I agree adding some info about needing access to our cloud.gov space before getting started. That probably ties in with the whole audience thing.

Maybe something at the start like:

>This section is primarily for 18F/TTS developers working on the Federalist project. Before you get started, make sure you have been fully on-boarded, including getting access to the Federalist cloud.gov space.

Reply:
I'm editing in the copy @jseppi wrote right after the next heading for this section (per the comment thread above this).